packages/core/src/app/shipping/ShippingForm.spec.tsx

```diff
@@ -358,4 +368,117 @@ describe('ShippingForm Component', () => {
             });
         });
     });
+
+    describe('Braintree AXO', () => {
+        afterEach(() => {
+            jest.clearAllMocks();
+        });
```

Comment:
I think this one can be moved to the first describe level (below beforeEach function)

Reply:
good point, thanks